src/components/Card.js

```diff
@@ -51,8 +77,8 @@ export default class Card {
     this.cardLocation = this._cardElement.querySelector(".card__location");
     // get card view
     this._cardImage.src = this._link;
-    this._cardImage.alt = this._location;
-    this.cardLocation.textContent = this._location;
+    this._cardImage.alt = this._name;
+    this.cardLocation.textContent = this._name;
     //set event listeners
```

Comment:
Needs correction: don't forget to render the initial likes using `this._renderLikes` method